src/server/cmd/pachd/main.go

```diff
@@ -84,6 +84,7 @@ type appEnv struct {
 	LogLevel              string `env:"LOG_LEVEL,default=info"`
 	IAMRole               string `env:"IAM_ROLE,default="`
 	ImagePullSecret       string `env:"IMAGE_PULL_SECRET,default="`
+	NoExposeDockerSocket  bool   `env:"NO_EXPOSE_DOCKER_SOCKET,default=false"`
```

Comment:
would it make sense to make this `EXPOSE_DOCKER_SOCKET` with the default set to `true`? Then you could have `if exposeDockerSocket` rather than `if !noExposeDockerSocket` later on

Reply:
I can definitely see the logic for that, I did it this way because throughout the code base `bool` values always default to `false` since that's what go naturally defaults them to and so we invert a lot of conditions so that the default value of `false` corresponds to the behavior we think makes for a better default. In this case it doesn't really make sense for this struct, but I think it does make sense for the feature as a whole, because the flag to enable it at deploy time is `--no-expose-docker-socket` if the flag was `--expose-docker-socket` with a default of true, then the only use for the flag would be `--expose-docker-socket=false` which I think is a little more confusing. This is pretty hairsplitty though.